docs/src/api/params.md

```diff
@@ -310,26 +310,32 @@ Target URL.
 
 Query parameters to be sent with the URL.
 
-## java-fetch-params
+## csharp-fetch-option-params
+* langs: csharp
+- `params` <[Object]<[string], [any]>>
```

Comment:
Why `any` and not only primitive types like in other languages?

Reply:
Because .NET does not allow unions, having it IDictionary<string, object> seems to be the best?